src/utils/utils.cairo

```diff
@@ -165,13 +155,28 @@ namespace Helpers {
         return (res=quotient);
     }
 
-    func compute_half_uint256{range_check_ptr}(val: felt*, i: felt, res: felt) -> (res: felt) {
-        if (i == 1) {
-            return (res=res + [val]);
+    // @notice: This function returns a numbers in the base b system
+    // @param digits: array of digets
+    // @param digits_len: length od digets
+    // @return res: decimal representation of digits
+    func compose_felt{range_check_ptr}(digits: felt*, digits_len: felt, base: felt) -> (
```

Comment:
please use `digits_len: felt*, digits: felt*, base: felt` in the signature (`list_len` before `list`)

Reply:
Done